agent/src/main.cpp

```diff
@@ -250,8 +250,10 @@ int main(int argc, char **argv)
     } catch (ConfigError &error) {
         log_error() << "Could not load configuration";
         return 1;
-    } catch (ReturnCode failure) {
-        log_error() << "Agent initialization failed";
+    } catch (SoftwareContainerError &error) {
+        // Currently only case where any other exception can occur is during Agent init
```

Comment:
I don't understand this comment. Is it a comment about the fact that we are catching an exception that is not config related?